From 244b3fe8fa7f2906a1a39550d864627ef6551a3d Mon Sep 17 00:00:00 2001
From: Eugen Hristev <eugen.hristev@microchip.com>
Date: Tue, 4 Jun 2019 11:01:12 +0300
Subject: [PATCH] dependency: remove HaveModule(qml)

In order to build XmlListModel plugin, we have to remove dependency to qml
module.
This is done in meta-qt5 in yocto project.
Thus, applying this patch here

Signed-off-by: Eugen Hristev <eugen.hristev@microchip.com>
---
 src/imports/imports.pro | 2 +-
 src/src.pro             | 6 ++----
 2 files changed, 3 insertions(+), 5 deletions(-)

diff --git a/src/imports/imports.pro b/src/imports/imports.pro
index 72046db..8bc3423 100644
--- a/src/imports/imports.pro
+++ b/src/imports/imports.pro
@@ -1,3 +1,3 @@
 TEMPLATE = subdirs
 
-qtHaveModule(qml): SUBDIRS += xmllistmodel
+SUBDIRS += xmllistmodel
diff --git a/src/src.pro b/src/src.pro
index 5083270..66035c7 100644
--- a/src/src.pro
+++ b/src/src.pro
@@ -1,8 +1,6 @@
 TEMPLATE = subdirs
 SUBDIRS +=  xmlpatterns
 
-qtHaveModule(qml){
-    SUBDIRS += imports
-    imports.depends = xmlpatterns
-}
+SUBDIRS += imports
+imports.depends = xmlpatterns
 
-- 
2.21.0

